commands/kubernetes_test.go

```diff
@@ -677,110 +698,137 @@ func TestKubernetesNodePool_Create(t *testing.T) {
 	})
 }
 
+func createTestNodePoolUpdateRequest() godo.KubernetesNodePoolUpdateRequest {
+	return godo.KubernetesNodePoolUpdateRequest{
+		Name:   testNodePool.Name,
+		Count:  &testNodePool.Count,
+		Tags:   testNodePool.Tags,
+		Labels: map[string]string{},
+	}
+}
+
 func TestKubernetesNodePool_Update(t *testing.T) {
-	// by cluster ID
-	withTestClient(t, func(config *CmdConfig, tm *tcMocks) {
-		r := godo.KubernetesNodePoolUpdateRequest{
-			Name:  testNodePool.Name,
-			Count: &testNodePool.Count,
-			Tags:  testNodePool.Tags,
-		}
-		tm.kubernetes.EXPECT().UpdateNodePool(testCluster.ID, testNodePool.ID, &r).Return(&testNodePool, nil)
+	t.Run("by cluster ID", func(t *testing.T) {
```

Comment:
As a drive-by improvement, I wrapped the test cases into `t.Run()` calls as I found it super hard to tell which tests failed in the old structure.

The individual tests did not change fundamentally. Let me know though if you'd like to me to split the `t.Run()`-related modification out into a separate commit.

Reply:
I think this makes sense for now. If that changes in the future, we can always revisit it. 
Thanks for the clarification!